index.html

```diff
@@ -1,12 +1,128 @@
+    </nav>
+  </header>
+  <main>
+    <h1>Дизайн-студия «nёrd's»</h1>
+    <!-- h1 визуально будет скрыт,необх для чтения поисковиками и скрин ридерами -->
+    <section>
+      <h2>Долго, дорого, скрупулёзно.</h2>
+      <a href="blank.html">Узнать больше</a>
+    </section>
+    <section>
+      <div>
+        <h2>Веб-сайты</h2>
+        <p>Мир ни когда не будет прежним после того как увидит ваш сайт!</p>
+        <a href="catalog.html">заказать</a>
+      </div>
+      <div>
+        <h2>Приложения</h2>
+        <p>Покорите топ-10 приложений в AppStore и Google Play</p>
+        <a href="catalog.html">заказать</a>
+      </div>
+      <div>
+        <h2>Презентации</h2>
+        <p>Вы даже не подозреваете, на сколько вы изумительны!</p>
+        <a href="catalog./html">заказать</a>
+      </div>
+    </section>
+    <section>
+      <h2>Мы маленькая, но гордая дизайн-студия из Краснодара.</h2>
+      <p>Исповедуем принципы минималтизма и чистоты. Ставим математический расчет превыше креатива. Работаем не покладая рук, как роботы.</p>
+      <h3>Выполняем заказы на разработку:</h3>
+      <ul>
+        <li>Веб-сайтов любой сложности</li>
+        <li>Мобильных приложений для IOS и Android</li>
+        <li>Слайдшоу и видео для корпоративных презентаций</li>
+      </ul>
+    </section>
+    <section>
+      <div>
+        <img src="img/nerds-illustration.png" width="360" height="208" alt="Год рождения Дизайн-студии «nёrd's»">
+      </div>
+      <h3>С 2004 года любим точность во всем:</h3>
+      <dl>
+        <dt>Уровень самоотдачи</dt>
+        <dd>146<sup>%</sup></dd>
+        <dt>Соблюдение сроков</dt>
+        <dd>100<sup>%</sup></dd>
+        <dt>Минимальная предоплата</dt>
+        <dd>50<sup>%</sup></dd>
+      </dl>
+    </section>
+    <section>
+      <div>
+        <a href="https://htmlacademy.ru/intensive/htmlcss">
+          <img src="#" width="199" height="68" alt="Сайт htmlacademy">
+        </a>
+      </div>
+      <div>
+        <a href="#">
+          <img src="#" width="209" height="90" alt="Сайт BORODINSKI">
+        </a>
+      </div>
+      <div>
+        <a href="#">
+          <img src="#" width="185" height="52" alt="Сайт PINK">
+        </a>
+      </div>
+      <div>
+        <a href="#">
+          <img src="#" width="173" height="84" alt="Сайт MISHKA">
+        </a>
+      </div>
+    </section>
+  </main>
+  <footer>
+    <section>
+      <div>
+        <a href="#">Вконтакте</a>
+      </div>
+      <div>
+        <a href="#">Фейсбук</a>
+      </div>
+      <div>
+        <a href="#">Инстаграм</a>
+      </div>
+    </section>
+    <section>
+      <h3>Давайте дружить, это выгодно!</h3>
+          <p>Скидка 10% для друзей из социальных сетей.</p>
+    </section>
+
+    <section>
+      <!--Здесь будет форма отпрвки сообщения "Напишите нам"-->
+      191186, Санкт-Петербург,
+      ул. Б. Конюшенная, д. 19/8
+      тел. <a href="tel:+7 (812) 275-75-75">+7 (812) 275-75-75</a>

```

Comment:
```suggestion
      тел. <a href="tel:+78122757575">+7 (812) 275-75-75</a>
```

![image](https://user-images.githubusercontent.com/17790771/93351815-d0a18e00-f842-11ea-8d83-a432897f1c98.png)